examples/use_completions.rs

```diff
@@ -0,0 +1,69 @@
+use schemars::JsonSchema;
+use serde::Deserialize;
+use serde::Serialize;
+
+use openai_safe::{
+    llm_models::{AnthropicModels, MistralModels, OpenAIModels},
+    Completions,
+};
+
+#[derive(Deserialize, Serialize, JsonSchema, Debug, Clone)]
+struct TranslationResponse {
+    pub spanish: String,
+    pub french: String,
+    pub german: String,
+    pub polish: String,
+}
+
+#[tokio::main]
+async fn main() {
+    env_logger::init();
+
+    // Example context and instructions
+    let instructions =
+        "Translate this exact English sentence to all the languages in the response type";
+
+    // Get answer using OpenAI
+    let openai_api_key: String = std::env::var("OPENAI_API_KEY").expect("OPENAI_API_KEY not set");
+    let model = OpenAIModels::Gpt4; // Choose the model
+
+    let openai_completion = Completions::new(model, &openai_api_key, None, None);
+
+    match openai_completion
+        .get_answer::<TranslationResponse>(instructions)
+        .await
+    {
+        Ok(response) => println!("OpenAI response: {:?}", response),
+        Err(e) => eprintln!("Error: {:?}", e),
+    }
+
+    // Get answer using Anthropic
```

Comment:
You will notice that no matter what LLM provider we use the code is exactly the same.